backend/src/services/v2/model.ts

```diff
@@ -206,6 +206,20 @@ export async function updateModelCard(
   return revision
 }
 
+export type UpdateModelParams = Pick<ModelInterface, 'name' | 'description' | 'visibility'>
+export async function updateModel(user: UserDoc, modelId: string, diff: Partial<UpdateModelParams>) {
```

Comment:
Does the 'diff' stand for different/difference??

Reply:
I would assume so, a Partial is (like the name suggests) a type that contains _some_ but not all of the properties of a given type. 

I think we should probably rename this, but I am going to approve in case you're 100% set on this naming @a3957273